LFA.qml

```diff
@@ -676,7 +687,12 @@ Item {
 
         Rectangle {
             x: -135
-            y: 205 - ((root.watertempf - 31) * 0.70192)
```

Comment:
Thank you for this. The math hurt my soul.

Reply:
:) Haha...yeah took me a few to get right.